tests/settings.py

```diff
@@ -126,5 +126,6 @@
 
 DEBUG_TOOLBAR_CONFIG = {
     # Django's test client sets wsgi.multiprocess to True inappropriately
-    "RENDER_PANELS": False
+    "RENDER_PANELS": False,
+    "IS_RUNNING_TESTS": False,
```

Comment:
Maybe add a comment here?

```suggestion
    # IS_RUNNING_TESTS must be False even though we're running tests because we're running the toolbar's own tests.
    "IS_RUNNING_TESTS": False,
```